### ISSUE TITLE
"Overview of the skore UI" doesn't really talk about UI

### ISSUE THREAD

**Issue by adrinjalali** [probabl-ai/skore#738]
This page (https://probabl-ai.github.io/skore/latest/auto_examples/plot_02_overview_skore_ui.html) doesn't really talk about UI. It's more of an exploration of the API. In effect, this content would ideally be in the API docs themselves.

**Comment by sylvaincom** [comment]
Yes, I changed the title of the example, and we will create an epic issue for the API
